---
title: crAssphage project
output: 
    github_document:
      toc: true
      toc_depth: 3
      fig_height: 8
      fig_width: 10
      dev: jpeg
bibliography: crAssphage.bib
---


# Background
Supplementary data analysis for the paper:\br
_**Karkman, A., Pärnänen, K., Larsson, DGJ.** 2018._
_Fecal pollution explains antibiotic resistance gene abundances in anthropogenically impacted environments._

## Introduction
Discharge of treated sewage introduces antibiotic resistance genes (ARGs) and resistant bacteria (ARBs) to the environment [@karkman_antibiotic-resistance_2018]. It has been speculated that the resistance determinants could be selected and/or disseminated to other bacteria in the downstream environments due to the antibiotcs and other selective agents released in the effluents [@rizzo_urban_2013; @guo_metagenomic_2017]. However, the increased abundance of ARGs in downstream environments could as well be explained by the amount of fecal pollution without any large scale selection and/or dissemination of the genes.  

Recently a human feacal phage, crAssphage, was discovered from human fecal metagenomes [@dutilh_highly_2014] and was shown to infect _Bacteroides intestinalis_ [@shkoporov_crass001_2018]. It has been shown to be very abundant in human fecal material and mostly specific to humans [@garcia-aljaro_determination_2017]. Due to these facts, it has already been used as fecal marker in various studies [@stachler_correlation_2018;@ahmed_precipitation_2018;@stachler_metagenomic_2014;@stachler_quantitative_2017;@ahmed_evaluation_2018]. Another fecal Bacteroides phage, ɸB124-14, has also been used for microbial source tracking. Unlike crAssphage, it should be abundant in porcine and bovine guts as well [@ogilvie_resolution_2017]. However, we did not find it to perform as well as crAssphage and it won't be included in the analyses. 

In this study we show that in most of the the studied environments the abundance of ARGs, *intI1* integrase gene and mobile genetic elements correlates well with fecal pollution levels with no evident signs of selection or dissemination of the resistance genes. The only exception being sediments polluted with wastewater from drug manufacturing containing exceptionally high levels of antibiotics [@bengtsson-palme_shotgun_2014;@kristiansson_pyrosequencing_2011].


# Bioinformatics
The bioinformartics part shows only example commands and is not meant to be run as such. The results from the bioinformatics paret are available in the `data` folder and will be used in the datraanalysis part with R.  

All metagenomic samples were downloaded from public repositories as described in the methods. 

The crAssphage ([NC_024711.1](https://www.ncbi.nlm.nih.gov/nuccore/NC_024711.1)) and ɸB124-14 ([HE608841.1](https://www.ncbi.nlm.nih.gov/nuccore/HE608841.1)) genomes were downloaded from
GenBank as fasta files. 

## Phage genomes and Bowtie index
The phage genomes were indexed using `bowtie2-build`
```{bash, eval=FALSE}
bowtie2-build phage_genome.fasta phage_genome
```
## Mapping reads against phage genomes and calculating genome coverage
After indexing the phage genomes each sample was mapped against the genomes. The average genome coverage was calculated and used as a proxy for phage abundance.  
__Paired-end reads:__
```{bash, eval=FALSE}
bowtie2 -x phage_genome -1 Sample_R1_reads.fastq.gz -2 Sample_R2_reads.fastq.gz -S Sample.sam
samtools view -Sb -f 2 Sample.sam > Sample.bam
samtools sort Sample.bam -o Sample_sort.bam
samtools index Sample_sort.bam
export GEN_COV=$(samtools depth -a Sample_sort.bam |\
                  awk '{ sum += $3; n++ } END { if (n > 0) print sum / n; }')
echo 'Sample\t'$GEN_COV
```
__Single reads:__
```{bash, eval=FALSE}
bowtie2 -x phage_genome -U Sample.fastq -S Sample.sam
samtools view -Sb -q 10 Sample.sam > Sample.bam
samtools sort  Sample.bam -o  Sample_sort.bam
samtools index Sample_sort.bam
export GEN_COV=$(samtools depth -a Sample_sort.bam |\
                  awk '{ sum += $3; n++ } END { if (n > 0) print sum / n; }')
echo 'Sample\t'$GEN_COV 
```

## ARG and MGE abundance
The `fastq` files were converted to `fasta` and the sample name was added to each sequence header before concatenating all fasta files to one file for annotation with DIAMOND.  

The count tables are generated from the DIAMOND outputs using custom scripts. The scripts for single (`parse_diamond.py`) and paired-end reads (`parse_diamondPE.py`)  can be downloaded from [here](https://github.com/karkman/parse_diamond).

__Paired-end reads:__
```{bash, eval=FALSE}
diamond blastx -d ResFinder -q Sample_R1.fasta --max-target-seqs 1 -o Sample_R1_res.txt \
                -f 6 --id 90 --min-orf 20 -p 24 --masking 0
diamond blastx -d ResFinder -q Sample_R2.fasta --max-target-seqs 1 -o Sample_R2_res.txt \
                -f 6 --id 90 --min-orf 20 -p 24 --masking 0
python parse_diamondPE.py -1 Sample_R1_res.txt -2 Sample_R2_res.txt -o Sample_ResFinder.csv
```

__Single reads:__
```{bash, eval=FALSE}
diamond blastx -d ResFinder -q Sample.fasta --max-target-seqs 1 -o Sample_res.txt \
                -f 6 --id 90 --min-orf 20 -p 24 --masking 0
python parse_diamondPE.py -i Sample_res.txt -o Sample_ResFinder.csv
```

To bp count in each metagenome was used for normalization. 

# Data analysis and statistics in R
The results from mapping against crAssphage and gene annotations were imported to R. 

## Load in the data and libraries needed in the analyses
All the data can be found from the `data` folder.
```{r, message=FALSE, eval=FALSE}
library(tidyverse)
HMP <- read.table("data/")
crass_imapct <- read.table("data/")
crass_wwtp <- read.table("data/")
crass_MG_RAST <- read.table("data/")
res_risk <- read.table("data/")
```
## Figure 1 - crAssphage in human fecal metagenomes 
text here
```{r, eval=FALSE}
par(fig=c(0,0.45,0,0.8), new=TRUE)
plot(log10(rel_res)~log10(rel_crAss), data=HMP, bg=cols[as.factor(HMP$country)], pch=21,
      ylab = "Normalized ARG abundance (log10)", xlab="Normalized crAssphage abundance (log10)", cex=2, ylim=c(2.5, 4.5))
par(fig=c(0,0.45,0.5,1), new=TRUE)
boxplot(log10(rel_crAss)~country, data=HMP, horizontal=TRUE, col=cols, axes=F)
axis(2, at=1:3, labels=c("China", "Europe", "US"), las=1)
title("A", adj = 0, line = 0)

par(fig=c(0.45,0.9,0,0.8), new=TRUE)
tmp <- subset(HMP, rel_int>0)
plot(log10(rel_res)~log10(rel_int), data=tmp, bg=cols[as.factor(tmp$country)], pch=21,
      ylab = "", xlab="Normalized intI1 abundance (log10)", cex=2, ylim=c(2.5, 4.5))
par(fig=c(0.45,0.9,0.5,1), new=TRUE)
boxplot(log10(rel_int)~country, data=tmp, horizontal=TRUE, col=cols, axes=F)
axis(2, at=1:3, labels=c("China", "Europe", "US"), las=1)
title("B", adj = 0, line = 0)

par(fig=c(0.8,1,0,0.8),new=TRUE)
boxplot(log10(rel_res)~country, data=HMP, col=cols, axes=F)
axis(1, at=1:3, labels=c("China", "Europe", "US"), las=3)
```

## Figure 2 - crAssphage in impacted environments
text here
```{r, eval=FALSE}
ggplot(crass_impact, aes(x=rel_crAss, y=rel_res, color=country)) +  geom_smooth(method="lm") + geom_point(aes(shape=crAss_detection), size=5) + scale_x_log10() + scale_y_log10() + theme_classic() +
            labs(y = "Normalized ARG abundance", x="Normalized crAssphage abundance", color="Study", shape="crAssphage detection") + scale_colour_manual(values=cols)
```

## Figure 3 - crAsspahge in MG-RAST metagenomes
text here.
```{r, eval=FALSE}
ggplot(MG_RAST_crass, aes(x=rel_crAss, y=rel_res, color=revised_source)) + geom_point(size=5) + scale_x_log10() + scale_y_log10() + geom_smooth(method="lm") + theme_classic() +
      labs(y = "Normalized ARG abundance", x="Normalized crAssphage abundance", color = "Revised source") + scale_colour_manual(values=cols)
```

# References